src/main/java/wolox/training/models/Book.java

```diff
@@ -65,82 +77,6 @@ public Book(String genre, String author, String image, String title, String subt
     public Book() {
```

Comment:
You can use @NoArgsConstructor and delete this method

Reply:
It was fixed at last commit